src/chatClient.cpp

```diff
@@ -3243,6 +3243,7 @@ promise::Promise<void> GroupChatRoom::invite(uint64_t userid, chatd::Priv priv)
 promise::Promise<void> GroupChatRoom::autojoinPublicChat(uint64_t ph)
 {
     Id myHandle(parent.mKarereClient.myHandle());
+    mAutoJoiningCompleted = false;
```

Comment:
If the request fails, we need to set the flag back to `true`, right?

Reply:
Fixed